Buildings/Resources/ReferenceResults/Dymola/Buildings_Air_Systems_SingleZone_VAV_Examples_Guideline36.txt

```diff
@@ -8,14 +8,14 @@ statistics-initialization=
 statistics-simulation=
 {
   "linear": "11, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0",
-  "nonlinear": "1",
+  "nonlinear": "2",
```

Comment:
I noted the increase in the dimension of the nonlinear system. Looking at dsmodel.mof, it seems to be due to different tearing. I don't think there is anything we can do to avoid it.